Documentation/Signoffs/Housing.md

```diff
@@ -0,0 +1,98 @@
+# Launcher Housing
+## Functionality 
+The launcher housing will provide power to the other subsystems connected to the main interceptor body, provide a functional E-Stop button, have a lighting system, and have a firing alarm. The power system will be made up of two power supplies, a 36 V to 12 V step down converter, a low level 5 V 2 channel relay, 4 power distribution blocks, and a latching twist to turn on push button. Lighting will be done with two 1 meter long LED strips in purple and gold colors. A 12 V buzzer will be used for the firing alarm.  
+## Constraints 
+
+* C5: Include a clearly marked power switch on the interceptor.
+    * In order to verify this constraint is met the interceptor must have a switch or switches that turn the interceptor on and off when flipped. 
+* C6: Include a clearly marked emergency stop option on the interceptor.  
+    * To verify this constraint is met the interceptor must have a labeled E-Stop that will prevent power from flowing to all non essential components of the interceptor when pressed. 
+* C10: Design an interceptor with lights that are visible when powered on.   
+    * The customer asked for decorative lighting that serves no purpose other than aesthetics. 
+* C12: Design an interceptor which plays an alert noise before firing. The noise played must be louder than 40 dB when within a 10 foot radius of the interceptor. 
+    * The customer asked for an alert to be played before the interceptor fired. A sound 40 dB or louder when within 10 feet of the device was chosen as 40 dB is a moderately loud sound [13] and 10 feet is larger than the game board. 

```

Comment:
Better. But the source isn't reputable. Let's try to find a better information source regarding how loud a warning signal aught to be. try google scholar.